### PR TITLE
add switch-project command

### LINKED ISSUE #2940: Add neuro config switch-project command

### PR DESCRIPTION
**Pull request by zubenkoivan**
closes #2940

### CODE REVIEW

**Review by YevheniiSemendiak — APPROVED** [review]
Awesome job! :muscle: 

Just some minor suggestions.

File: CLI.md [hunk 1]
```diff
@@ -73,6 +73,7 @@
 		* [neuro config aliases](#neuro-config-aliases)
 		* [neuro config docker](#neuro-config-docker)
 		* [neuro config get-clusters](#neuro-config-get-clusters)
+		* [neuro config get-projects](#neuro-config-get-projects)
```
Comment (dalazx) [inline CLI.md:76]: should this command be under `admin` where all related commands are?

Comment (dalazx) [reply CLI.md:76]: ah, there is one already. missed it.
could you please remind why we need the two separate commands?

**Review by dalazx — APPROVED** [review]
looks great

File: CLI.md (hunk 1 above)
Comment (zubenkoivan) [reply CLI.md:76]: There is already one in admin section, removed this from config.

**Pull request merged**